themes/qgis/assets/css/custom.css

```diff
@@ -87,4 +91,14 @@ p {
 .follow-qgis {
   border-left-style: solid;
   border-left-color: white;
+}
+
+/* styling for search box */
+.search-input {
+  padding: .5em;
+  border: solid 2px #d6d3d3;
+}
+
+.search-input:focus {
+  outline: 1px solid;
 }
```

Comment:
No changes should be needed to this file if you follow my comments on menu.html

Reply:
@rcoffie -final comment for now: Your changes break the tests - please fix the tests before asking for an updated review.